src/containers/b2b/RequisitionList.tsx

```diff
@@ -148,12 +180,31 @@ class RequisitionList extends Component<CartCreateProps, CartCreateState> {
   }
 
   handleSaveList() {
-    const { listName } = this.state;
+    const { listName, createRequisitionForm } = this.state;
     if (listName.length === 0) {
       this.setState({ listNameErrorMessages: intl.get('name-is-required') });
     } else {
-      this.setState({ listNameErrorMessages: '' });
-      this.handleModalClose();
+      cortexFetch(createRequisitionForm.self.uri, {
+        method: 'post',
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: localStorage.getItem(`${Config.cortexApi.scope}_oAuthToken`),
+        },
+        body: JSON.stringify({
+          name: listName,
```

Comment:
The key name was changed lately.
```suggestion
          itemListName: listName,
```